README.md

```diff
@@ -1 +1,51 @@
-# styleguide
+
+# Ombulabs::Styleguide
```

Comment:
@FionaDL Please follow our style guide for this: http://ombulabs.github.io/styleguide/

It should be "OmbuLabs::StyleGuide"

Reply:
Ah yes, I see.